spacexbt/eliza
# Improve default character configuration

This PR includes several improvements to the default character configuration:

1. Changed default model provider from LLAMALOCAL to OPENAI for better out-of-box experience
2. Removed duplicate "How do you handle stress?" entries
3. Made character responses more consistent with the sophisticated persona
4. Improved message examples to be more aligned with character bio
5. Streamlined adjectives list to focus on intellectual traits

Testing Instructions:
1. Run the starter project with default configuration
2. Verify character responses are consistent
3. Check that all message examples work as expected
